iml-gui/crate/src/components/command_modal.rs

```diff
@@ -188,25 +274,26 @@ pub(crate) fn view(model: &Model) -> Node<Msg> {
     }
 }
 
-fn command_item_view(x: &Command, open: bool) -> Node<Msg> {
-    let border = if !open {
+fn command_item_view(model: &Model, x: &RichCommand) -> Node<Msg> {
+    let is_open = is_typed_id_selected(&model.select, TypedId::Command(x.id));
+    let border = if !is_open {
         C.border_transparent
-    } else if x.cancelled {
-        C.border_gray_500
-    } else if x.errored {
-        C.border_red_500
     } else if x.complete {
         C.border_green_500
+    } else if x.errored {
+        C.border_red_500
+    } else if x.cancelled {
+        C.border_gray_500
```

Comment:
This ordering needs to be reverted. All commands will be marked complete when they finished, it's the absence of other states that makes them successful.

Reply:
I see, thanks.